File: app/common/src/main/java/com/example/common/model/ChiTietDonHang.java
package com.example.common.model;

import com.google.gson.annotations.SerializedName;

public class ChiTietDonHang {
    @SerializedName("_id")
    private String _id;

    @SerializedName("MaDonHang")
    private String maDonHang;

    @SerializedName("MaSanPham")
    private String maSanPham;

    @SerializedName("SoLuong")
    private int soLuong;

    @SerializedName("DonGia")
    private double donGia;

    @SerializedName("ChietKhau")
    private double chietKhau;

    @SerializedName("ThanhTien")
    private double thanhTien;

    public String get_id() { return _id; }
    public void set_id(String _id) { this._id = _id; }

    public String getMaDonHang() { return maDonHang; }
    public void setMaDonHang(String maDonHang) { this.maDonHang = maDonHang; }

    public String getMaSanPham() { return maSanPham; }
    public void setMaSanPham(String maSanPham) { this.maSanPham = maSanPham; }

    public int getSoLuong() { return soLuong; }
    public void setSoLuong(int soLuong) { this.soLuong = soLuong; }

    public double getDonGia() { return donGia; }
    public void setDonGia(double donGia) { this.donGia = donGia; }

    public double getChietKhau() { return chietKhau; }
    public void setChietKhau(double chietKhau) { this.chietKhau = chietKhau; }

    public double getThanhTien() { return thanhTien; }
    public void setThanhTien(double thanhTien) { this.thanhTien = thanhTien; }
}

File: app/quan_ly_thong_tin_ca_nhan/src/main/java/com/example/quan_ly_thong_tin_ca_nhan/quanlytaikhoan/api/ChiTietDonHang.kt
package com.example.quan_ly_thong_tin_ca_nhan.quanlytaikhoan.api

import com.google.gson.annotations.SerializedName

data class ChiTietDonHang(
    @SerializedName("_id")
    val _id: String? = null,

    @SerializedName("MaDonHang")
    val maDonHang: String? = null,

    @SerializedName("MaSanPham")
    val maSanPham: String? = null,

    @SerializedName("SoLuong")
    val soLuong: Int? = null,

    @SerializedName("DonGia")
    val donGia: Double? = null,

    @SerializedName("ChietKhau")
    val chietKhau: Double? = null,

    @SerializedName("ThanhTien")
    val thanhTien: Double? = null
)
